config/docker/thinking_sphinx.yml

```diff
@@ -1,44 +1,43 @@
-common: &sphinx
-  configuration_file: <%= ENV.fetch('THINKING_SPHINX_CONFIGURATION_FILE') { Rails.root.join("config/#{Rails.env}.sphinx.conf")} %>
+production:
+  # ----------------------------------
+  # Thinking Sphinx configuration
+  # see https://gist.github.com/pat/a7d73376dd657b4457092efc9e9c418a
+  # ----------------------------------
+  mysql41: 9306
```

Comment:
different port number than original

Reply:
It's not different, it's just explicit, rather than falling back to the default value, which is [9306](https://gist.github.com/pat/a7d73376dd657b4457092efc9e9c418a#file-thinking_sphinx-yml-L101).

Maybe we can just drop this setting, I just preferred it to be explicit.

Reply:
Ah, ok, I saw the `test` setting.

Reply:
The previous value was

`mysql41:            <%= 9313 + ENV['TEST_ENV_NUMBER'].to_i %>`

How is this equivalent to 9306? Was `TEST_ENV_NUMBER` always equal to `-7`?

I don't understand this part

Reply:
@jlledom That value you referenced is for the `test` environment. I believe this is to be able to run multiple tests in parallel and avoid ports clashing.

For tests, the config is still there in `config` directory:
https://github.com/3scale/porta/blob/317cb833f9/config/thinking_sphinx.yml#L27

In `thinking_sphinx.yml` under `config/docker` I have eliminated the `test` environment completely, keeping only `production` (and the idea is to remove `config/docker` folder completely, as it doesn't bring any value, started it here: https://github.com/3scale/porta/pull/3107).